### DISCUSSION TITLE
Relevanssi – A Better Search

### DISCUSSION THREAD

**Discussion by osman-kocak** [inspireui/support#5523] [Working on Features]
Hello, can you integrate this plugin into the mobile app?

url [https://tr.wordpress.org/plugins/relevanssi/](https://tr.wordpress.org/plugins/relevanssi/)

**Comment by minhcasi** [comment]
Latest version is supported the Ajax Search Pro

**Reply by osman-kocak** [reply]
Ajax Search Pro just show first image its not show second or other images on fluxstore woocommerce

**Comment by officialsuyogdixit** [comment]
Better to directly integrate with FacetWP, as it also supports routing search queries through Relevanssi. With this, there will be a unified interface. Without Relevanssi, the search will be slow with 100000+ products with 200+ ACF custom metafields(not only Attributes) filled on each product, which is in my case.

**Comment by usamaahmed5656** [comment]
you can use elastic press plugin with elastic search if you are not on a shared hosting

**Comment by officialsuyogdixit** [comment]
ElasticPress does not support Facets. We have more than 200 custom meta fields of products on which we wish to have facet search. ElasticPress facets only filter by taxonomy currently.

Kindly support FacetWP, as it also supports routing search queries through Relevanssi. We will buy this immediately if this is done.

**Comment by usamaahmed5656** [comment]
@suyogricha  elastic press does support facets ..not only facets it also supports [Synonyms]. the latest release has all the features you will need for search ...
